network/definition/discrete_values.yaml

```diff
@@ -6,6 +6,23 @@ switchStatus:
   "OFF": 0
   "ON": 1
 
+inputStatus:
+  "FALSE": 0
+  "TRUE": 1
+
+ENG_gear:
+  "GEAR_UNKNOWN": 0
+  "GEAR_1": 1
+  "GEAR_2": 2
+  "GEAR_3": 3
+  "GEAR_4": 4
+  "GEAR_5": 5
+  "GEAR_N": 6
+
+ENG_state:
+  "ENGINE_STATE_STALL": 1
+  "ENGINE_STATE_CRANKING": 2
+  "ENGINE_STATE_RUNNING": 3
```

Comment:
Remove these, these are from IC and we dont use that, also their names should have been engGear...